java/src/main/java/ai/rapids/cudf/CuFileBuffer.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.rapids.cudf;
+
+/**
+ * Represents a cuFile buffer.
+ */
+public final class CuFileBuffer implements AutoCloseable {
```

Comment:
These new native resource tracking classes should be hooked into the `Cleaner` interface so we can track when they leak (and try to automatically clean them up when they do leak).

Reply:
Done.